digdag-cli/src/main/java/io/digdag/cli/Profile.java

```diff
@@ -0,0 +1,127 @@
+package io.digdag.cli;
+
+import com.beust.jcommander.IStringConverter;
+import com.beust.jcommander.Parameter;
+import com.fasterxml.jackson.datatype.guava.GuavaModule;
+import com.fasterxml.jackson.module.guice.ObjectMapperModule;
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import io.digdag.cli.profile.TaskAnalyzer;
+import io.digdag.cli.profile.TasksSummary;
+import io.digdag.client.api.JacksonTimeModule;
+import io.digdag.client.config.Config;
+import io.digdag.client.config.ConfigElement;
+import io.digdag.core.DigdagEmbed;
+import io.digdag.core.config.ConfigModule;
+import io.digdag.core.config.PropertyUtils;
+import io.digdag.core.database.DatabaseModule;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
+import static io.digdag.cli.SystemExitException.systemExit;
+import static io.digdag.client.DigdagClient.objectMapper;
+
+
+public class Profile
+    extends Command
+{
+    @Parameter(
+            names = {"--from"},
+            required = true,
+            converter = TimestampConverter.class
```

Comment:
It might be helpful if `description` is defined with timestamp format.

Reply:
Ah, unfortunately digdag-cli doesn't use this field at all and outputs the descriptions of options like https://github.com/treasure-data/digdag/pull/1533/files#diff-cee923b77be1e7ada3d52d17f3d385a391e14e6870f9328a1d235b9f5d44ba9eR120-R124.

I noticed other 3 options have the meaningless field and I'll remove them.